rest/data/interface.go

```diff
@@ -235,4 +235,5 @@ type Connector interface {
 	FindCommitQueueByID(string) (*restModel.APICommitQueue, error)
 	CommitQueueRemoveItem(string, string) (bool, error)
 	CommitQueueClearAll() (int, error)
+	IsAuthorized(context.Context, string, string, string) (bool, error)
```

Comment:
can we replace the three string arguments with an arg struct? 

Also the name is unclear, authorized for what? by who?

Reply:
Done.